src/app/recipe/new/page.tsx

```diff
@@ -0,0 +1,79 @@
+import cc from "classcat";
+
+import { Edit, Plus } from "@/components/icons";
+import { ImageCarousel, ImageComponent } from "@/components/image";
+import { AddButton } from "@/app/recipe/new/_components/AddButton";
+import { Header } from "@/app/recipe/new/_components/Header";
+import { IngredientListItem } from "@/app/recipe/new/_components/IngredientListItem";
+import { RecipeStepItem } from "@/app/recipe/new/_components/RecipeStepItem";
+import { newRecipeData } from "@/app/recipe/new/data";
+
+const Page = () => {
+  const TitleClass = cc(["mb-1 ml-4 font-bold text-black"]);
```

Comment:
コンポーネントまたは定数として持ちたいなど特殊な状況を除き、キャメルケースで統一したいです！
classcatは条件でCSSを出し分ける時などに使うためのPKGなので使いまわしただけの場合は使用しなくて良いかなと思います。

```suggestion
  const titleClass = "mb-1 ml-4 font-bold text-black";
```